make is easy to configure this via inventory

Natkeeran opened the pull request:
# What does this Pull Request do?
Make the urls as variable so they can be configured more easily in the inventory.

# How should this be tested?
* Replace the role with the PR
* Vagrant up
* Verify the things get index in Fedora


# Interested parties
@Islandora-Devops/committers

Natkeeran committed:
make is easy to configure this via inventory

whikloj commented on defaults/main.yml:
```diff
@@ -33,13 +39,13 @@ alpaca_settings:
       input.stream: activemq:topic:fedora
       triplestore.reindex.stream: activemq:queue:triplestore.reindex
-      triplestore.baseUrl: http://localhost:8080/bigdata/namespace/islandora/sparql
+      triplestore.baseUrl: "{{ alpaca_triplestore_base_url }}/namespace/islandora/sparql"
```
If we are going part way, why not remove the assumption that all triplestores will use the "islandora" namespace.

whikloj commented on defaults/main.yml:
```diff
@@ -25,6 +25,12 @@ alpaca_karaf_deploy_dir: "{{ alpaca_karaf_dir }}/deploy"
+alpaca_milliner_base_url: http://localhost:8000/milliner/
+alpaca_gemini_base_url: http://localhost:8000/gemini/
+alpaca_houdini_base_url: http://localhost:8000/houdini/convert
+alpaca_homarus_base_url: http://localhost:8000/homarus/convert
```
Feels like the name `..._base_url` might be incorrect for these last two.

Natkeeran replied:
@whikloj endpoint?

whikloj commented on defaults/main.yml:
```diff
@@ -25,6 +25,12 @@ alpaca_karaf_deploy_dir: "{{ alpaca_karaf_dir }}/deploy"
 alpaca_karaf_user: karaf
 alpaca_local_mvn_path: /opt/maven/repo
 
+alpaca_triplestore_base_url: http://localhost:8080/bigdata
```
Probably should be consistent on whether to end with a slash or not. Unless we can adjust to account for either

Natkeeran committed:
Update main.yml

Natkeeran committed:
Update main.yml

Natkeeran commented:
@whikloj 

Addressed your feedback. 

Triplestore url is used here as well: https://github.com/Islandora-Devops/ansible-role-blazegraph.  Will update that role as well.

Natkeeran committed:
Update main.yml

dannylamb commented:
`vagrant up`

dannylamb reviewed (approved):
Tested along with the other PRs, and this works as intended.  e.g. nothing broke but now we can use all these variables.
